types/burns/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for burns 2.0
+// Project: https://github.com/shalvah/burns#readme
+// Definitions by: Timothy Onyiuke <https://github.com/timolinn>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const _Burns: Burns;
+
+export interface ObjectLiteral {
+    [key: string]: any;
+}
+
+export interface Config {
+    defaultHandler?: any;
+    broadcaster?: any;
+    pusher?: any;
+}
+
+declare class Burns {
+    configure(config?: Config): this;
+
+    registerEvents(newEvents: ObjectLiteral): this;
```

Comment:
TypeScript already provides a built-in `Record` type that satisfies this:

```suggestion
    registerEvents(newEvents: Record<string, any>): this;
```